tests/unittest/_torch/attention/sparse/test_cute_dsl_fp8_paged_mqa_logits.py

```
@@ -0,0 +1,753 @@
+                dsl_schedule_meta = get_paged_mqa_logits_metadata(
+                    data["context_lens"], compute_block_kv, num_sms
+                )
+
+                def dsl_fn(data=data):
+                    torch.ops.trtllm.cute_dsl_fp8_paged_mqa_logits(
+                        data["q_fp8"],
+                        data["kv_fused"],
+                        data["weights"],
+                        data["context_lens"],
+                        data["block_table"],
+                        dsl_schedule_meta,
```

Ruff (0.15.12)

[warning] Function definition does not bind loop variable `dsl_schedule_meta`

(B023)